dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -198,15 +197,22 @@ def sync_course
     unless Policies::Lti.roster_sync_enabled?(current_user)
       return redirect_to home_path
     end
-    params.require([:lti_integration_id, :deployment_id, :context_id, :rlid, :nrps_url]) if params[:section_code].blank?
+
+    if params[:section_code].blank?
+      begin
+        params.require([:lti_integration_id, :deployment_id, :context_id, :rlid, :nrps_url])
+      rescue ActionController::ParameterMissing => _exception
+        return render_sync_course_error("Attempting to sync a course or section from the wrong place.", :bad_request, "wrong_context")
```

Comment:
Super nit, there are a mixture of single and double quotes on these new changes (here and below). Our [style guide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#ruby) prefers single quotes for non-interpolated strings.